lib/insert/insert_donation.rb

```diff
@@ -173,7 +173,7 @@ def self.insert_payment(kind, fee_total, donation_id, data)
   def self.insert_donation(data, entities)
     d = Donation.new
     d.date = data['date']
-    d.anonymous = data['anonymous']
+    d.anonymous = data['anonymous'].nil? ? false : true
```

Comment:
We could add the `:before_save` I did before callback instead, I'm not sure which one would be better.

Reply:
Oops, I just realized this is not going to work, I'm going to fix it

Reply:
Okay, I'm going to change for `:before_save` then!